code/modules/clothing/outfits/ert.dm

```diff
@@ -460,3 +460,47 @@
 		/obj/item/gun/ballistic/automatic/sniper_rifle=1,\
 		/obj/item/gun/grenadelauncher/security=1,\
 		/obj/item/gun/ballistic/automatic/ar=1)
+
+/datum/outfit/hitman
+	name = "Hitman"
+
+	uniform = /obj/item/clothing/under/suit/black
+	shoes = /obj/item/clothing/shoes/sneakers/black
+	gloves = /obj/item/clothing/gloves/color/black
+	ears = /obj/item/radio/headset
+	glasses = /obj/item/clothing/glasses/sunglasses/advanced
+	l_hand = /obj/item/storage/secure/briefcase
+	id = /obj/item/card/id/syndicate
+	belt = /obj/item/pda/heads
+	r_hand = /obj/item/storage/box/syndie_kit/chameleon
```

Comment:
why not just start them out wearing chameleon clothes so they dont have to switch out?